modules/ROOT/pages/to-create-an-asset.adoc

```diff
@@ -27,6 +27,18 @@ This illustration summarizes how each asset type (in green) appears in Exchange:
 
 image::ex2-exchange-assets.png[Illustration - How Assets appear in Exchange]
 
+== Asset Limits
+
+Exchange uses an asset limit to prevent accidental or intentional denial of service by uploading too many assets.
+
+The limit is 500 assets for organizations with trial accounts, and 100,000 assets for other organizations.
```

Comment:
I would change `organizations` to `master organizations` because the limit is reached for master organizations, not for a business group

Reply:
Fixed. Thank you!